packages/webdriverio/src/commands/element/nextElement.ts

```diff
@@ -22,7 +24,7 @@
  * @type utility
  */
 
-export default function nextElement (this: WebdriverIO.Element) {
+export default function nextElement (this: WebdriverIO.Element): ChainablePromiseElement<any>  {
```

Comment:
Is this needed? TS figures this out automatically? Also we should avoid `any` if possible

Reply:
When ts figuring it out it generates `import("../..").ChainablePromiseElement<WebdriverIO.Element>`, and this path causes problems in node16/nodenext module resolution.
I admit that I have to replace `any` with `WebdriverIO.Element`.